scripts/mutect2_wdl/mutect2_multi_sample.wdl

```diff
@@ -35,6 +35,7 @@ workflow Mutect2 {
 	File cosmic
```

Comment:
Can you put the description of the multi-pair tsv file format at the top of the WDL?